boards/shields/sparkfun_carrier_asset_tracker/doc/index.rst

```diff
@@ -0,0 +1,118 @@
+.. _semtech_sx1276mb1mas:
```

Comment:
copy paste error / this needs to be unique to this shield

Reply:
you are right, done!